app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+  <div class="medium-4 columns">
+    <div>
```

Comment:
Same here, and below as well -- we should remove these unless we need them for something.